src/ops/ops.go

```diff
@@ -151,6 +151,40 @@ func (o *ops) removeNodeImage(liveLogger io.Writer) error {
 	return nil
 }
 
+func (o *ops) applyInstallKargs(liveLogger io.Writer, installerArgs []string) error {
+	commandArgs := []string{"kargs", "--os", installStateRootName}
+
+	lastArgIndex := len(installerArgs) - 1
+	for i, arg := range installerArgs {
+		if arg == "--append-karg" && i < lastArgIndex {
+			commandArgs = append(commandArgs, "--append", installerArgs[i+1])
+			continue
+		}
+		if arg == "--delete-karg" && i < lastArgIndex {
+			commandArgs = append(commandArgs, "--delete-if-present", installerArgs[i+1])
+			continue
+		}
+	}
```

Comment:
Feels like we should error out if `--append-karg` or `--delete-karg` _is_ last rather than do our best to work with it.

Reply:
I _think_ we have a validation around this in assisted (where the user adds these). And if we don't I'd rather it fail there than here.